.github/workflows/e2e-tests.yaml

```
@@ -73,3 +73,21 @@ jobs:
             retention-days: 10
             path: |
               **/*.log
+
+        - name: login to quay (e2e test image build)
+          id: podman-login-quay
+          if: success()
+          run: |
+            podman login --username ${{ secrets.QUAY_ODH_KUBERAY_TESTS_PASSWORD }} --password ${{ secrets.QUAY_ODH_KUBERAY_TESTS_USERNAME }} quay.io
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)